src/api-service/__app__/timer_daily/__init__.py

```diff
@@ -7,15 +7,30 @@
 
 import azure.functions as func
 from onefuzztypes.enums import VmState
+from onefuzztypes.events import EventProxyCreated
 
-from ..onefuzzlib.events import get_events
+from ..onefuzzlib.events import get_events, send_event
 from ..onefuzzlib.proxy import Proxy
 from ..onefuzzlib.webhooks import WebhookMessageLog
 from ..onefuzzlib.workers.scalesets import Scaleset
 
 
 def main(mytimer: func.TimerRequest, dashboard: func.Out[str]) -> None:  # noqa: F841
     for proxy in Proxy.search():
+        if proxy.is_outdated():
+            logging.info("marking proxy as outdated.")
+            proxy.outdated = True
+            proxy.save()
+            if (
+                len(Proxy.search(query={"region": [proxy.region], "outdated": [False]}))
+                == 0
+            ):
+                logging.info("outdated proxy, creating new one.")
```

Comment:
Please log more detail here, such as the region

Reply:
Done.